iot-hub/Quickstarts/ReadD2cMessages/Parameters.cs

```diff
@@ -15,19 +15,19 @@ internal class Parameters
         [Option(
             'e',
             "EventHubCompatibleEndpoint",
-            HelpText = "The event hub-compatible endpoint from your IoT Hub instance. Use `az iot hub show --query properties.eventHubEndpoints.events.endpoint --name {your IoT Hub name}` to fetch via the Azure CLI.")]
+            HelpText = "The event hub-compatible endpoint from your IoT hub instance. Use `az IoT hub show --query properties.eventHubEndpoints.events.endpoint --name {your IoT hub name}` to fetch via the Azure CLI.")]
```

Comment:
We are using "iot" in lowercase for Azure CLI in the [docs](https://docs.microsoft.com/en-us/cli/azure/iot/hub?view=azure-cli-latest), should we align this here?

Reply:
yep, mistake to change the cli text.